### PR TITLE
SIDM-3582 SIDM-3404 Account Activation link expired screen

### PR DESCRIPTION
**Pull request by dfourn**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3582
https://tools.hmcts.net/jira/browse/SIDM-3404

### Change description ###

Due to the preview build problems, I've combined the 2 tickets into 1 PR to avoid merge conflicts later on. 

Included missing welsh translations for the SIDM-3353 as well.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -35,15 +35,12 @@ public.user.activation.title.text=User Activation
 public.user.activation.password.one=Create your password
 public.user.activation.password.two=Repeat password 
 public.user.activation.expired.title=Link Expired
-public.user.activation.expired.text.p1.1=To get a new link, you need to
-public.user.activation.expired.text.p1.2=create your account again
-public.user.activation.expired.text.p2=You can use the same email and password.
+public.user.link.expired.heading=Your link has expired, or has already been used
+public.user.activation.expired.text.p1=To create an account, you need to start this process again by submitting a new Create Account request. You can use the same email address as before.
+public.user.activation.expired.text.p1.hyperlink=To create an account, you need to start this process again by submitting a new <a href="{0}">Create Account</a> request. You can use the same email address as before.
```
Comment (tbamido) [inline src/main/resources/messages.properties:40]: how will this changes affect welsh translations?

File: src/test/java/uk/gov/hmcts/reform/idam/web/UserControllerTest.java [hunk 2]
```diff
@@ -512,4 +514,16 @@ public void selfRegister_shouldPopulateTheModelWithTheUsersDetailsIfCalledWithAV
...
+     * @verifies return null if redirecturi or clientid are empty
+     * @see UserController#buildRegistrationLink(ActivationResult)
+     */
+    @Test
+    public void buildRegistrationLink_shouldReturnNullIfRedirecturiOrClientidAreEmpty() throws Exception {
+        UserController userController = new UserController();
+        assertNull(userController.buildRegistrationLink(new ActivationResult()));
+        assertNull(userController.buildRegistrationLink(new ActivationResult().redirectUri(GOOGLE_WEB_ADDRESS)));
+        assertNull(userController.buildRegistrationLink(new ActivationResult().clientId(CLIENT_ID)));
+    }
```
Comment (tbamido) [inline src/test/java/uk/gov/hmcts/reform/idam/web/UserControllerTest.java:528]: not sure this is the best way to test a controller

if you follow the pattern in use in this test file you can use mockMvc and test whatever is using the `buildRegistrationLink` method

Comment (dfourn) [reply src/test/java/uk/gov/hmcts/reform/idam/web/UserControllerTest.java:528]: I think this is better than using mocks.

Comment (dfourn) [reply src/test/java/uk/gov/hmcts/reform/idam/web/UserControllerTest.java:528]: @tbamido let me know if you're okay with it staying as a normal unit test

**Pull request merged**